shared/wallets/wallet/index.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/wallets'
+import {Box2, Divider, SectionList, Text} from '../../common-adapters'
+import Header from './header-container'
+import Asset from '../asset/container'
+import {globalColors, globalMargins, styleSheetCreate} from '../../styles'
+
+type Props = {
+  accountID: Types.AccountID,
+  sections: any[],
+}
+
+export default (props: Props) => {
+  const renderItem = ({item, index, section}) => {
+    const children = []
+    if (section.title === 'Your assets') {
+      children.push(
+        <Asset accountID={props.accountID} index={item.item} key={`${props.accountID}:${item.item}`} />
+      )
+      if (index !== section.data.length - 1) {
+        // don't put divider after last thing in section
+        children.push(<Divider key={`${props.accountID}:${item.item}:divider`} />)
+      }
+    }
+    // TODO
```

Comment:
(what's the TODO here?)

Reply:
to render the transaction rows, I'll remove it in the second PR